system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -1,15 +1,15 @@
-.top-sites-list {
-  $top-sites-size: $grid-unit;
-  $top-sites-border-radius: 6px;
-  $top-sites-title-height: 30px;
-  $top-sites-vertical-space: 8px;
-  $screenshot-size: cover;
-  $rich-icon-size: 96px;
-  $default-icon-wrapper-size: 42px;
-  $default-icon-size: 32px;
-  $default-icon-offset: 6px;
-  $half-base-gutter: $base-gutter / 2;
+$top-sites-size: $grid-unit;
```

Comment:
This type of change is one of the main reason to keep various refactoring to their own commit. These indentation changes makes it incredibly difficult to review functionality changes when there's a ton of unrelated changes mixed in.

Reply:
Sorry about that, I can take this into it's own commit.

Reply:
I think I've worked through most of it manually comparing with two windows for side-by-side to figure out what was just indentation / moving up/down vs what's changes.